client/components/domains/register-domain-step/index.jsx

```diff
@@ -31,19 +32,53 @@ var wpcom = require( 'lib/wp' ).undocumented(),
 	abtest = require( 'lib/abtest' ).abtest;
 
 // max amount of domain suggestions we should fetch/display
-var SUGGESTION_QUANTITY = 10,
+const SUGGESTION_QUANTITY = 10,
 	INITIAL_SUGGESTION_QUANTITY = 2;
 
+const analytics = analyticsMixin( 'registerDomain' );
+
+let searchQueue = [],
+	searchStackTimer = null;
+
+function processSearchQueue() {
+	var q = searchQueue.slice();
```

Comment:
`q` is not exactly the most descriptive variable name :wink: Also, `const` instead of `var`? :)